utils/kata-manager.sh

```diff
@@ -23,6 +23,8 @@ readonly containerd_slug="containerd/containerd"
 readonly kata_project_url="https://github.com/${kata_slug}"
```

Comment:
Adding my comment here, but this is for the commiot message. Just reading the commit message, the usage of filename fragments is not really clear to me. Could we reword it?

Reply:
Fixed.